### PR TITLE
RBGO-112: Add script to parse the swagger file in a repeatable way

### PR DESCRIPTION
**Pull request by ron-bierman-unity**
This PR reworks some of the documentation framework. The following changes have been made:
- Added documentation on how to regenerate the API documentation
- The `layout.erb` has been changed to put the includes at the top
- The introduction and flows are now regular includes and no longer depend on `erb` logic
- The toc creation is split up in two sections. See screenshot
Screen Shot 2021-04-26 at 12 06 38 PM

### CODE REVIEW

File: source/includes/_introduction.md.erb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+#### SnapChance API supports users with three roles:
+- <b>Regulator</b> - Create and manage the licenses.
+- <b>Operator/Admin</b> - Create and manage the lotteries.
+- <b>Player</b> - Add funds to the wallet; purchases and play tickets.
+
+Base URLs:
+
+* <a href="https://api-stg3.snapchance.no">https://api-stg3.snapchance.no</a>
+
+Web: <a href="https://snapchance.no">SnapChance</a>
```
Comment (ron-bierman-unity) [inline source/includes/_introduction.md.erb:16]: This is just a plain copy of the original text. I will update this in a separate PR

File: source/index.html.md [hunk 2]
```diff
@@ -0,0 +1,12885 @@
+---
```
Comment (ron-bierman-unity) [inline source/index.html.md:1]: FYI. This file is autogenerated

File: source/layouts/layout.erb [hunk 3]
```diff
@@ -83,7 +85,24 @@ under the License.
...
+                  </li>
+                <% end %>
+              </ul>
+            <% end %>
+          </li>
+        <% end %>
+        <li>
+          <span class="toc-divider">Endpoints</span>
+        </li>
+        <% toc_data(api_content).drop(1).each do |h1| %>
```
Comment (ron-bierman-unity) [inline source/layouts/layout.erb:105]: We `drop(1)` here to remove the `Snapchance v2.0.0` title

File: scripts/convertSwaggerToMarkdown.js [hunk 4]
```diff
@@ -5,30 +5,39 @@ const YAML = require('yaml')
 const options = {
   codeSamples: true,
   httpsnippet: false,
-  language_tabs: [{csharp: "C#"}, {shell: "Shell"}, {http: "HTTP"}, {'javascript': 'JavaScript'}, {'javascript--node': 'Node.JS'}, {ruby: "Ruby"}],
+  language_tabs: [
+    { 'csharp': "C#" },
```
Comment (justinhamadeunity) [inline scripts/convertSwaggerToMarkdown.js:9]: 👍

File: package.json [hunk 5]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "fotolotto-snapchance-api-docs",
+  "version": "1.0.0",
+  "description": "Documentation for the multilot application platform",
+  "main": "scripts/convertSwaggerToMarkdown.js",
+  "scripts": {
+    "convert" : "node scripts/convertSwaggerToMarkdown"
```
Comment (justinhamadeunity) [inline package.json:7]: Would it make sense to wrap other functions in NPM scripts?

File: source/includes/_introduction.md.erb (hunk 1 above)
Comment (justinhamadeunity) [reply source/includes/_introduction.md.erb:16]: 👍

**Review by justinhamadeunity — APPROVED** [review]
Looks good, comments are minor.  Feel free to merge.

File: source/includes/_introduction.md.erb (hunk 1 above)
Comment (justinhamadeunity) [reply source/includes/_introduction.md.erb:16]: @dkaga note that this moved and is not in the swagger doc anymore

File: source/layouts/layout.erb (hunk 3 above)
Comment (justinhamadeunity) [reply source/layouts/layout.erb:105]: @dkaga do you have time to help with styling?

**Pull request merged**